src/features/defi/providers/foxy/components/FoxyManager/Deposit/FoxyDeposit.tsx

```diff
@@ -95,6 +95,9 @@ export const FoxyDeposit = () => {
       [DefiStep.Approve]: {
         label: translate('defi.steps.approve.title'),
         component: Approve,
+        props: {
+          contractAddress,
+        },
```

Comment:
Not a fan of a catch-all prop called props, especially not typed as `Record<any, any>` but that works for the purpose of fixing this